Truth or Truth/Screens/Settings/SettingsCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  SettingsCell.swift
+//  Truth or Truth
+//
+//  Created by eli dangerfield on 8/3/20.
+//  Copyright © 2020 Josiah Rininger. All rights reserved.
+//
+
+import UIKit
+
+final class SettingsCell: UICollectionViewCell {
+    
+    var imageView = UIImageView.create(with: UIImage())
+    let titleLabel = UILabel.create(fontSize: 22, numberOfLines: 1, textAlignment: .center, isBold: true)
+    
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+        
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupView() {
+        addSubviews(imageView, titleLabel)
+        setupConstraints()
+    }
+    
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            imageView.leftAnchor.constraint(equalTo: leftAnchor),
+            imageView.topAnchor.constraint(equalTo: topAnchor),
+            imageView.bottomAnchor.constraint(equalTo: bottomAnchor),
+            imageView.widthAnchor.constraint(equalToConstant: 40),
+            
+            titleLabel.leftAnchor.constraint(equalTo: imageView.rightAnchor, constant: 10),
```

Comment:
`titleLabel.leadingAnchor.constraint(equalTo: imageView.trailingAnchor, constant: 10),`
We almost always will use leading/trailing instead of left/right anchors. This because some countries read right to left and thus the content should be laid out right to left. Using the leftAnchor constrains the View to the left but leadingAnchor constrains the View to what is defined as leading in the country. In general, iOS Devs always use leading and trailing unless they find a specific reason not to. The image [here](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/SupportingRight-To-LeftLanguages/SupportingRight-To-LeftLanguages.html) is a dope visual of this.